### DISCUSSION TITLE
maxWidths dont work

### DISCUSSION THREAD

**Discussion by gportail** [wire-elements/modal#182] [Q&A]
It seem there is a problem in the definition of `protected static array $maxWidths` in _LivewireUI\Modal\ModalComponent_ (_vendor/wire-elements/modal/src/ModalComponent.php_)

I don't use TailwindCSS but I redefine all `.sm:max-w-xxx` class in my own CSS
In v 1.0.3 it work perfectly
in v1.0.4 the width of modals is not good.

I correct `$maxWidths` with:
```php    
protected static array $maxWidths = [
        'sm'  => 'sm:max-w-sm',
        'md'  => 'sm:max-w-md',
        'lg'  => 'sm:max-w-lg md:max-w-lg',
        'xl'  => 'sm:max-w-xl md:max-w-xl',
        '2xl' => 'sm:max-w-2xl md:max-w-2xl lg:max-w-2xl',
        '3xl' => 'sm:max-w-3xl md:max-w-3xl lg:max-w-3xl',
        '4xl' => 'sm:max-w-4xl md:max-w-4xl lg:max-w-4xl xl:max-w-4xl',
        '5xl' => 'sm:max-w-5xl md:max-w-5xl lg:max-w-5xl xl:max-w-5xl',
        '6xl' => 'sm:max-w-6xl md:max-w-6xl lg:max-w-6xl xl:max-w-6xl 2xl:max-w-6xl',
        '7xl' => 'sm:max-w-7xl md:max-w-7xl lg:max-w-7xl xl:max-w-7xl 2xl:max-w-7xl',
    ];
```

The dialogs are the rigth size with this value

**Comment by gportail** [comment]
maybe but I don't use TailwindCSS ... as I said

**Comment by spekkie2002** [comment]
You're right, i had a brainfart here, sorry...

**Comment by emiliosh** [comment]
Oh thanks, I thought I was crazy.
